commands/volumes/common.go

```diff
@@ -0,0 +1,125 @@
+package volumecommands
+
+import (
+	"os"
+	"strings"
+
+	"github.com/gluster/glusterd2/gdctx"
+	"github.com/gluster/glusterd2/servers/sunrpc"
+	"github.com/gluster/glusterd2/transaction"
+	"github.com/gluster/glusterd2/utils"
+	"github.com/gluster/glusterd2/volgen"
+	"github.com/gluster/glusterd2/volume"
+	"github.com/gluster/glusterd2/xlator"
+
+	"github.com/pborman/uuid"
+)
+
+func areOptionNamesValid(optsFromReq map[string]string) bool {
+
+	var xlOptFound bool
+	for o := range optsFromReq {
+
+		// assuming option to be of the form <domain>.<xlator-option>
+		// and <domain> will be the xlator type.
+		// Example: cluster/afr.eager-lock
```

Comment:
We should in the future allow options with just the xlator name as the domain, ie. allow `afr.eager-lock` as well instead of just `cluster/afr.eager-lock`, to make this a little more user friendly.